stdlib/sql/vertica.go

```diff
@@ -42,7 +42,11 @@ func (m *VerticaRowReader) GetNextRow() ([]values.Value, error) {
 	row := make([]values.Value, len(m.columns))
 	for i, col := range m.columns {
 		switch col := col.(type) {
-		case bool, int, uint, int64, uint64, float64, string:
+		case int:
+			row[i] = values.NewInt(int64(col))
+		case uint:
+			row[i] = values.NewUInt(uint64(col))
+		case bool, int64, uint64, float64, string:
 			row[i] = values.New(col)
```

Comment:
Nit: Should we add support for `int` and `uint` in the `values.New()` function? That would allow us to to handle all of these types in one case. Maybe there's a good reason why we don't do that already, but I'm not sure what it is.

Reply:
I had the same idea, but @jsternberg encouraged the change be limited to the vertica-related code (at least for now).

It does seem to me like widening values that are too narrow should be safe enough in all cases. It also seems like any database driver could give 32-bit numbers based on the pre-existing schema, depending on how the DDL was authored.

Maybe Jonathan can speak to his rationale.

We did talk about perhaps updating `values.New()` to error instead of what amounts to swallowing and converting to a nil, but that sort of change could have wide impact and should come with some time to evaluate the size first.

Reply:
I'm not sure it would necessarily be bad, but there was a similar section in the influxql code where we had a type switch on an interface and chose a point type for it. We originally had `uint` get translated to integers and it was a bit complicated when we wanted to add support for unsigned types because it was a breaking change in that API. The reason why I was thinking to keep it constrained and avoid any casts is just to avoid any potential ambiguities in that core part of the API.

Reply:
Briefly touched on this with Nathaniel just now, and another aspect is how we might like to consider `values.New()` as a convenience only appropriate for "stuff that was already vetted as Flux values." The implication being any use in places like the `sql` package would be "illegal" and we should always be using the constructors for specific Value variants in this case.

I don't know what it'd take to make `values.New()` private to only things that _should be free to use it_, whoever that might be. Alternatively, maybe we add a docstring to coach the usage. :shrug: